code/modules/organs/limbs.dm

```diff
@@ -1131,7 +1131,12 @@ treat_grafted var tells it to apply to grafted but unsalved wounds, for burn kit
 	return ((status & LIMB_BROKEN) && !(status & LIMB_SPLINTED))
 
 /obj/limb/proc/is_malfunctioning()
-	return ((status & (LIMB_ROBOT|LIMB_SYNTHSKIN)) && prob(brute_dam + burn_dam))
+{
+	if(status & LIMB_ROBOT)
+		return ((status & LIMB_ROBOT) && prob(brute_dam + burn_dam))
+	else if(status & LIMB_SYNTHSKIN && (brute_dam + burn_dam) > 10)
+		return (status & LIMB_SYNTHSKIN) && (prob(brute_dam + burn_dam))
+}
```

Comment:
```suggestion
	if(status & LIMB_ROBOT)
		return ((status & LIMB_ROBOT) && prob(brute_dam + burn_dam))
	else if(status & LIMB_SYNTHSKIN && (brute_dam + burn_dam) > 10)
		return (status & LIMB_SYNTHSKIN) && (prob(brute_dam + burn_dam))
```
also I don't think you need this **((status & LIMB_ROBOT)**

Reply:
How is the game going to know to trigger the malf proc on LIMB_ROBOT limbs? I'd rather not touch it as that part was the original proc.

Reply:
kk understandable